### ISSUE TITLE
WCAG22 Quickref - Missing links

### ISSUE THREAD

**Issue by KerstinProbiesch** [w3c/wcag#3589]
For some SC in the Quickref (https://www.w3.org/WAI/WCAG22/quickref/) the link "show techniques and failures for..." is not present: 

- SC 1.3.5, 
- 2.1.3, 
- 2.4.10 and
- 3.3.6

**Comment by fstrr** [comment]
I think this issue needs to be in the [Quickref repo](https://github.com/w3c/wai-wcag-quickref) repo.

**Comment by mbgower** [comment]
@KerstinProbiesch The Quick Ref material is not maintained by the AGWG. Would you please open this issue against the Quickref repo @fstrr links to above, and close this?

**Comment by fstrr** [comment]
[Duplicated the issue in the QuickRef repo](https://github.com/w3c/wai-wcag-quickref/issues/254), so am closing this.
